week1/MAKEME.md

```diff
@@ -0,0 +1,132 @@
+3. Inside `<Button>` create a `<button>` with the text `Add 1!` and `onClick` attribute
+4. Pass down the `count` state variable to `<Count>`, and the `setCount` to `<Button>`
+5. Inside `<Counter>`, declare a variable called `feedback` above the return statement. Give this variable a ternary operator value: if `count` is higher than 10 it should display the string `"It's higher than 10!"`, else it displays `"Keep counting..."`
+6. 5. Test out if it works by importing `<Counter>` into the top level component, which is `<App>`
+
+## **3. Code along**
+
+Learning how to work with React is a fundamentally different approach to building frontends. Therefore, we need to practice a lot!
+
+In the following tutorial you'll learn how to a `Random Quote Machine`!
+
+Notice how you'll be writing `class` based components. This is an older React feature for writing components, but that's still valid.
+
+Even though you'll be learning more modern ways of writing React applications, you should always be aware that applications can vary in the versions of technologies they're working with.
+
+Don't worry! You don't have to memorize it all. Aim for understanding the principle behind it to the best you can. If you ever feel in doubt: do a Google search or ask a question in Slack!
+
+Enjoy!
+
+- [Build a Random Quote Machine](https://www.youtube.com/playlist?list=PL3cz80ehFCan4srLIp1HmhrQ0CU8Cw9XM)
+
+## **4. PROJECT: Hack Your Weather I**
+
+> Before you start, create a new folder called `project` that includes the files for the following web application you'll be building.
+
+During this module you'll be starting a project called `Hack Your Weather`! Like in the Node.js module, each week you'll add to it until it becomes a fully-fledged application you can be proud of.
+
+Here's what you're eventually going end up with: [Hack Your Weather Finished](https://hyf-weather-app.netlify.com/)
+
+Let's get into it.
+
+### 1. Setup
+
+The first step to any project is to setup your development environment. Follow the steps:
+
+1. Navigate to your `project` folder
+2. Inside, use `create-react-app` to create a React application. Give it the name `hackyourweather`
+3. Copy and paste the file `city-weather.json` (you can find this inside of the `project-starter-files` folder) inside of your project folder: into the `src` folder
+
+### 2. Requirements
+
+This week you'll build the foundation of your application. It should eventually look like this:
+
+![Week 1 Wireframe](../assets/project/week1.png)
+
+Here are the requirements your project needs to fulfill:
+
+- List all the cities and their weather information (taken from `city-weather.json`)
+- Display each city in its own box
+- Each box should contain the following pieces of data:
+  - `city name`
+  - `country name`
+  - `weather main`
+  - `weather desciption`
+  - `max and min temprature`
+  - `location (longitude and lattitude)`
+
+Hints:
+
+- Make use of the `map()` function to show the
```

Comment:
Unfinished sentence: 'to show the ... different cities'?

Reply:
Indeed. Fixing it...